docs/user-guide_installation.md

```diff
@@ -252,22 +268,25 @@ Initial credentials:
 
 	- For Windows, you could try `Advanced IP Scanner`. Download the tool from [here](https://download.advanced-ip-scanner.com).
 
-	- For Linux, you can use nmap command:
-
-		```
+	- For Linux, you can use nmap command:  
+    ```
 		sudo apt-get install nmap #For installing Nmap
-		sudo nmap -sn 192.168.1.0/24 #For scanning IP address
+    ```  
+    ```
+    sudo nmap -sn 192.168.1.0/24 #For scanning IP address
 		```
 
+    Alternatively you may also determine the IP address in the DHCP status page of your DHCP server (often included in a router).
+
 	**Connect to DietPi via SSH**
 
-	- A popular SSH Client for Windows is PUTTY. You can download putty from [here](https://www.chiark.greenend.org.uk/~sgtatham/putty/latest.html). Enter in the `Host name` field the IP address found during the scanning, select `SSH` and then click on _Open_ button.
+	- A popular SSH Client for Windows is PUTTY. You can download putty from [here](https://www.chiark.greenend.org.uk/~sgtatham/putty/latest.html). Enter in the `Host Name` field the IP address found during the scanning, select `SSH` and then click on _Open_ button.
 	![DietPi-ssh](assets/images/dietpi-ssh.jpg)
 
-	- Most Linux distributions come packaged with an ssh client. Type in your Terminal next command (replace the sample IP address 192.168.1.20 with the one found via scanning the network).
-	```
-	ssh pi@192.168.1.20
-	```
+	- Most Linux distributions come packaged with an ssh client. Type in your Terminal next command (replace the sample IP address 192.168.1.20 with the one found via scanning the network):  
+    ```
+  	ssh root@192.168.1.20
+  	```
```

Comment:
Please take care tabs indentation. E.g. on GitHub (and e.g. my notepad++) tabs have up to 8 character width while in your editor (obviously 😉) they have up to 4 characters width, leading to inconsistent indentation. I think in Markdown we should stick which white space indentation and should omit tab characters completely. Also in HTML later those are not interpreted and we cannot be sure how MkDocs handles them.